docs/resources/rdb_instance.md

```diff
@@ -79,7 +79,7 @@ resource "scaleway_rdb_instance" "main" {
   private_network {
     pn_id  = scaleway_vpc_private_network.pn.id
     ip_net = "172.16.20.4/22"   # IP address within a given IP network
-    (enable_ipam = false)
+    # enable_ipam = false
```

Comment:
Maybe add a comment here.
If you use service_ip, the enable_ipam MUST be set to false.
Or if by default it will be false, let's omit it all together